test-requirements.txt

```diff
@@ -0,0 +1,16 @@
+git+https://github.com/juju/python-libjuju.git
```

Comment:
Is there any reason why we're leaving this to fetch from the upstream?

Reply:
And same for the package below

Reply:
In fact, I'm not even sure how it ended up here. I thought I removed it. 
It's gone now.

Reply:
Happens to me frequently haha